### ISSUE TITLE
RCS didn't fixed after last update 

### ISSUE THREAD

**Issue by AlirezaParsi** [chiteroman/PlayIntegrityFix#371]
--------- beginning of main
07-09 08:27:56.576  6897  6897 D PIF     : [cache_key.package_info]: 1505372362354534520
07-09 08:27:56.577  6897  6897 D PIF     : [cache_key.has_system_feature]: 1505372362354534135
07-09 08:27:56.594  6897  7436 D PIF     : [cache_key.package_info]: 1505372362354534520
07-09 08:27:56.596  6897  7436 D PIF     : [cache_key.has_system_feature]: 1505372362354534135
07-09 08:27:56.628  6897  7436 D PIF     : [cache_key.display_info]: 1505372362354534590
07-09 08:27:56.628  6897  7436 D PIF     : [cache_key.display_info]: 1505372362354534590
07-09 08:27:56.628  6897  7436 D PIF     : [cache_key.display_info]: 1505372362354534590
07-09 08:27:56.631  6897  7436 D PIF     : [cache_key.has_system_feature]: 1505372362354534135
07-09 08:27:56.653  6897  7436 D PIF     : [cache_key.package_info]: 1505372362354534520
07-09 08:27:56.654  6897  7436 D PIF     : [cache_key.package_info]: 1505372362354534520
07-09 08:27:56.654  6897  7436 D PIF     : [cache_key.package_info]: 1505372362354534520
07-09 08:27:56.683  6897  7436 D PIF     : [cache_key.package_info]: 1505372362354534520
07-09 08:27:56.690  6897  7436 D PIF     : [ro.surface_flinger.has_wide_color_display]: true
07-09 08:27:56.690  6897  7436 D PIF     : [ro.surface_flinger.has_HDR_display]: true
07-09 08:27:56.710  6897  7436 D PIF     : [ro.boot.flash.locked]: 1
07-09 08:27:56.710  6897  7436 D PIF     : [ro.boot.vbmeta.device_state]: locked
07-09 08:27:56.828  6897  7436 D PIF     : [persist.sys.pihooks.security_patch]: 2024-06-05
07-09 08:27:56.828  6897  7436 D PIF     : [persist.sys.pihooks.security_patch]: 2024-06-05
07-09 08:27:56.829  6897  7436 D PIF     : [cache_key.package_info]: 1505372362354534520
07-09 08:27:56.830  6897  7436 D PIF     : [cache_key.package_info]: 1505372362354534520
07-09 08:27:56.830  6897  7436 D PIF     : [cache_key.package_info]: 1505372362354534520
07-09 08:27:56.833  6897  7436 D PIF     : [cache_key.display_info]: 1505372362354534590
07-09 08:27:56.833  6897  7436 D PIF     : [cache_key.display_info]: 1505372362354534590
07-09 08:27:56.833  6897  7436 D PIF     : [cache_key.display_info]: 1505372362354534590
07-09 08:27:56.854  6897  7436 D PIF     : [ro.zygote]: zygote64_32
07-09 08:27:56.854  6897  7436 D PIF     : [persist.sys.pihooks.first_api_level]: 24
07-09 08:27:56.854  6897  7436 D PIF     : [persist.sys.pihooks.first_api_level]: 24
07-09 08:27:56.855  6897  7436 D PIF     : [ro.boot.verifiedbootstate]: green
07-09 08:27:56.855  6897  7436 D PIF     : [cache_key.has_system_feature]: 1505372362354534135
07-09 08:27:56.855  6897  7436 D PIF     : [cache_key.has_system_feature]: 1505372362354534135
07-09 08:27:56.857  6897  7436 D PIF     : [ro.secure]: 1
07-09 08:27:56.857  6897  7436 D PIF     : [ro.secure]: 1
07-09 08:27:56.857  6897  7436 D PIF     : [ro.debuggable]: 0
07-09 08:27:56.857  6897  7436 D PIF     : [ro.debuggable]: 0
07-09 08:27:56.857  6897  7436 D PIF     : [sys.usb.state]: mtp
07-09 08:27:56.863  6897  7436 D PIF     : [ro.boot.hardware.revision]: 9.19.0
07-09 08:27:56.867  6897  7436 D PIF     : [ro.board.first_api_level]: 24
07-09 08:27:56.867  6897  7436 D PIF     : [ro.board.first_api_level]: 24
07-09 08:27:56.867  6897  7436 D PIF     : [ro.vendor.api_level]: 24
07-09 08:27:56.867  6897  7436 D PIF     : [ro.vendor.api_level]: 24

---------------------
Device: poco f3 
Rom : aospa unofficial 
![Screenshot_20240709-083305_Key_Attestation](https://github.com/chiteroman/PlayIntegrityFix/assets/29276037/1c62cbc1-73fd-48e8-81f1-d4bbaa8d15ae)

**Comment by asdfjkluiop** [comment]
As noted on #338 the fingerprint currently provided publicly here is banned for RCS and only RCS. You have to provide a custom print if you want RCS to work.

**Comment by TTVert** [comment]
I really hope this gets fixed w/o having to provide a custom fingerprint.. Fingers crossed.

**Comment by asdfjkluiop** [comment]
I really hope so too but given the other issue was closed with won't fix I have a suspicion that chiteroman doesn't care about RCS and won't change the print until it gets banned outright, which is really unfortunate as I only use PIF for RCS, I really don't much care about anything else that google blocks. I suppose tbf the fact that a fingerprint is still provided at all is surprising given he said he was going to stop doing that several releases ago(and did for a brief time)

**Comment by hwittenborn** [comment]
Looks like [v16.6](https://github.com/chiteroman/PlayIntegrityFix/releases/tag/v16.6) was released? I'm not currently on a rooted device, if someone who *is* rooted would be willing to test it, it'd be appreciated :sweat_smile:

**Comment by asdfjkluiop** [comment]
@hwittenborn  IT WORKS!!!!!!!!!! I'm so happy

**Comment by TTVert** [comment]
You guys rock, thanks for all the hard work!

**Comment by FyrusCarmin** [comment]
It doesn't seem to work on my side, does it still works for you guys ?

**Comment by asdfjkluiop** [comment]
yes, it still works for me

**Comment by FyrusCarmin** [comment]
Do you need a custom Pif for it ? Or maybe I'm just not updated enough on my Xiaomi EU rom

**Comment by asdfjkluiop** [comment]
No, you do not need a custom PIF if you are running v16.6 or later
